packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -157,6 +157,7 @@ type Workspace {
     cursor: String
     filter: WorkspaceProjectsFilter
   ): ProjectCollection!
+  cost: Int!
```

Comment:
It it really just an integer? Not a structured object of price points and whatever? I don't understand what this is supposed to represent and other API consumers won't either - could you add a docblock to this field?

Reply:
Also - this might need @hasWorkspaceRole(ADMIN) on it, we probably don't want this to be shown to anyone else